subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -136,13 +139,21 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	/* TODO: start sending BIS empty data packet for each BIS */
 	ull_adv_iso_start(adv_iso, 0 /* TODO: Calc ticks_anchor */);
 
+	/* Associate the ISO instance with a Periodic Advertising and
+	 * an Extended Advertising instance
+	 */
+	lll_adv_iso = &adv_iso->lll;
+	lll_adv_sync->iso = lll_adv_iso;
+	lll_adv_iso->adv = &adv->lll;
+
 	/* Prepare BIG complete event */
 	/* TODO: Implement custom node_rx struct for optimization */
 	node_rx = (void *)&adv_iso->node_rx_complete;
 	node_rx->hdr.type = NODE_RX_TYPE_BIG_COMPLETE;
 	node_rx->hdr.handle = big_handle;
 	node_rx->hdr.rx_ftr.param = adv_iso;
 
+	/* FIXME: instead of is_created use lll_adv_iso->adv being set */
```

Comment:
Is anything stopping you from doing that in this PR? Sounds like a simple change.

Reply:
Will do.